### ISSUE TITLE
Static # of tweets returned ==> dynamic

### ISSUE THREAD

**Issue by Kellenbj** [mkovach2/EC601_proj2#1]
Consider taking user input to allow a non static number of tweets to be returned. This way if someone uses this code they will only get back the number of tweets they desire at that time. 
A simple approach:

numtweets = input("How many tweets would you like analyzed?")
lastTweet = tot.myOwnLast(numtweets)

To allow numtweets to be non static.

**Comment by mkovach2** [comment]
good idea.  I'll have to combine this functionality with the existing option, probably.  I'd like to leave the code modular in that it should be usable to agents which are not users with direct command-line access.  Thanks!
